modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -380,18 +424,53 @@ export default class ParticipantConnectionStatus {
             isConnActiveByJvb = true;
         }
 
-        const isConnectionActive
-            = isConnActiveByJvb
-                && (isVideoMuted || (isInLastN && !isVideoTrackFrozen));
+        let newState = ParticipantConnectionStatus.INACTIVE;
+
+        if (isConnActiveByJvb) {
+            if (isInLastN) {
```

Comment:
last N should be only an excuse to not going to INACTIVE if video is reported as frozen